bahmni-mart/files/bahmni-mart.json

```diff
@@ -0,0 +1,157 @@
+{
```

Comment:
@bhiravabhatla , I guess we should discuss about that. Is this a configuration layer that allows to define what entities to flatten or something?

Reply:
Yes, This is the config file which mart uses to filter which of the openmrs tables to flatten. By default, many jobs are configured to run, we can choose to have our own config - which should be part of bahmni_config. We can talk more about this.